massadmin/massadmin.py

```diff
@@ -387,7 +387,16 @@ def mass_change_view(
             obj=obj)
 
 
-class MassEditMixin:
-    actions = (
-        mass_change_selected,
-    )
+class MassEditMixin(object):
+
+    def get_actions(self, request):
+        actions = super(MassEditMixin, self).get_actions(request)
+
+        if settings.MASS_USERS_GROUP and settings.MASS_USERS_GROUP in [g.name for g in request.user.groups.all()]:
```

Comment:
Why do you use a group for this rather than checking permission with `has_perm()`?
It seems to me, that using permission would be much more flexible - the permission can be given to more than one groups.
You can also add permission to non-managed models so this functionality can work entirely without additional settings: https://stackoverflow.com/questions/13932774/how-can-i-use-django-permissions-without-defining-a-content-type-or-model

Reply:
I implemented a new "can perform mass editing" permission with a new proxy model as from stackoverflow answer.
everything seem fine.